examples/Viewer.tsx

```diff
@@ -114,12 +107,17 @@ interface CustomType {
     };
 }
 
-const simulariumController = new SimulariumController({});
+interface InputParams {
+    localBackendServer: boolean;
+    useOctopus: boolean;
+}
+
+let simulariumController;
```

Comment:
Looks like `simulariumController` can also safely become a class property of `Viewer`? If that doesn't make sense for some reason I'm missing, I'd still give it a type here: `let simulariumController: SimulariumController;`

Reply:
It appears you might be right, lemme take a look. I usually just try to change as little code as possible when dealing with TypeScript, but I'll investigate this one and change it to a class property unless I can figure out why that may not be possible. At the very least, I can certainly give it a type

- [ ] try making simulariumController a class variable, or at least give it a type

Reply:
I think that makes sense for this app but in general I think we wanted to emphasize that the SimulariumController could be initialized and live outside of the core Viewer (as in Model-View-Controller paradigm).  Also possibly mirroring (or being an example for) the initialization sequence of simularium-website?

Reply:
Honestly, the original reason I had to move the SimulariumController initialization was from when I had been sending useOctopus and localBackendServer directly to them, so I'll just move it back to where it was originally.

@toloudis do you know why we're importing SimulariumController from `"../es"`? Cameron and I were looking at the code yesterday afternoon and when I was trying to set `simulariumController`'s type, and my editor wasn't thrilled about it because the SimulariumController we're importing is already transpiled into javascript.

Reply:
Yes, the point of importing from `/es` is to simulate as if we were using the npm-installed version of the library.  Some issues can come up with the building and bundling of the distribution that would arise sooner from here before having to publish it.

Reply:
We should definitely export any types needed by clients of the viewer.

Reply:
Alright, that does make sense since it's an example viewer, good to test the whole. I'm still a bit novice when it comes to TypeScript, so just to make sure, does that mean it _doesn't_ make sense to try to type the simulariumController, like should I just leave if how it is?